### ISSUE TITLE
ble scna does not find any device

### ISSUE THREAD

**Issue by simonepreite** [esphome/bluetooth-proxies#65]
Hi,

I followed the procedure to configure a generic ESP32 using the online tool you provided.
Then I added the ESP32 to home assistant (which was correctly connected to my LAN) by clicking configure.
The problem is that I do not see any bluethooth device in my home assissant

Am I doing something wrong or missing a step?

Thanks

**Comment by balloob** [comment]
Nope, that's correct. Home Assistant will only discover Bluetooth LE devices that it supports.
